lib/stripe.js

```diff
@@ -251,6 +261,50 @@ Stripe.prototype = {
       this[utils.pascalToCamelCase(name)] = new resources[name](this);
     }
   },
+
+  _getPropsFromConfig(config) {
+    // config can be an object or a string
+    const isString = typeof config === 'string';
+    const isObject = config === Object(config) && !Array.isArray(config);
+
+    if (!isObject && !isString) {
```

Comment:
part of me thinks we should accept `null` as valid, and treat it as the same as `undefined` (that is, don't set `apiVersion`)